src/backend/src/routes/work-packages.routes.ts

```diff
@@ -34,6 +34,21 @@ workPackagesRouter.post(
   validateInputs,
   WorkPackagesController.createWorkPackage
 );
+workPackagesRouter.post(
+  '/template/create',
+  nonEmptyString(body('templateName')),
+  nonEmptyString(body('templateNotes')),
+  nonEmptyString(body('workPackageName').optional()),
+  isWorkPackageStageOrNone(body('stage').optional()),
+  intMinZero(body('duration')),
+  body('expectedActivities').isArray(),
+  body('deliverables').isArray(),
+  body('blockedBy').isArray(),
+  nonEmptyString(body('blockedBy.*.name')),
+  isWorkPackageStageOrNone(body('blockedBy.*.stage').optional()),
+  validateInputs,
+  WorkPackagesController.createWorkPackageTemplate
```

Comment:
you can remove all this code since it's not related to hard deleting material type